datastore/models.go

```diff
@@ -338,6 +338,7 @@ type GroupConfig struct {
 	Strategy                 *StrategyConfiguration        `json:"strategy"`
 	Signature                *SignatureConfiguration       `json:"signature"`
 	RetentionPolicy          *RetentionPolicyConfiguration `json:"retention_policy" bson:"retention_policy"`
+	MaxPayloadReadSize       uint64                        `json:"max_payload_read_size" bson:"max_payload_read_size"`
```

Comment:
@danvixent Let's change this to `MaxIngestSize`.

Reply:
@Subomi we also use it for response from endpoints after sending an event there. `MaxIngestSize` suggests it's only for ingesting events